scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContext.java

```diff
@@ -62,21 +62,28 @@
 
     private final String progressFilePath;
 
+    private final String nodeUrl;
+
+    private final String nodeName;
+
     public TaskContext(ExecutableContainer executableContainer, TaskLauncherInitializer initializer,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)